### PR TITLE
Fix 4.3 env file for the test deployment

### PR DESCRIPTION
**Pull request by SchSeba**
Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: hack/env.sh [hunk 1]
```diff
@@ -3,7 +3,11 @@ NAMESPACE=openshift-ptp
 OPERATOR_EXEC=oc
 
 export RELEASE_VERSION=v4.3.0
+export IMAGE_TAG=4.3
 export OPERATOR_NAME=ptp-operator
 
+# This will find the latest digest.
 LINUXPTP_DAEMON_IMAGE_DIGEST=$(skopeo inspect docker://quay.io/openshift/origin-ptp | jq --raw-output '.Digest')
-export LINUXPTP_DAEMON_IMAGE=${LINUXPTP_DAEMON_IMAGE:-quay.io/openshift/origin-ptp@${LINUX_DAEMON_IMAGE_DIGEST}}
+PTP_OPERATOR_IMAGE_DIGEST=$(skopeo inspect docker://quay.io/openshift/origin-ptp-operator | jq --raw-output '.Digest')
```
Comment (zshi-redhat) [inline hack/env.sh:11]: DIGEST env var can be removed if IMAGE_TAG is used.

Comment (SchSeba) [reply hack/env.sh:11]: I just want to leave it compatible with the master branch

**Comment by zshi-redhat** [comment]
@SchSeba do we still need this PR for CI? it requires a bz be associated.

**Comment by SchSeba** [comment]
let's close this one as it required a BZ and this only affects the CI.
If there is a request for this I will reopen it and create a Bugzilla for it

/close

**Pull request closed without merging**